users/config.rst

```diff
@@ -149,6 +169,11 @@ device
     Syncthing will currently add this automatically if it is not present in
     the configuration file.
 
+minDiskFreePct
+    The amount of space that should be available on the disk this folder
```

Comment:
"The percentage of space that..."
"amount of" implies that it is an amount like 100MB, 1GB... but the units of the name are in percent.
Also on the following line change another "amount" to "percentage"